.tours/1---welcome.tour

```diff
@@ -0,0 +1,30 @@
+{
+    "title": "1 - Welcome to Easy Spaces Sample App",
```

Comment:
Maybe add a line about what the app does.

Reply:
Good catch! Added!